emailproxy.py

```diff
@@ -279,6 +289,58 @@ def _load():
             AppConfig._GLOBALS = configparser.SectionProxy(AppConfig._PARSER, APP_SHORT_NAME)
         AppConfig._SERVERS = [s for s in config_sections if CONFIG_SERVER_MATCHER.match(s)]
         AppConfig._ACCOUNTS = [s for s in config_sections if '@' in s]
+
+        def fetch_aws_secrets():
+            if 'boto3' not in sys.modules:
+                Log.error('Error: client configuration for some accounts includes an aws_secret'
+                          ' parameter, but the app has been loaded without the --aws-secrets option.')
+                raise ValueError("--aws-secrets must be specified if account configuration includes aws_secret")
+
+            # Create dict of AWS Secret IDs across all accounts
+            aws_secrets = dict.fromkeys([AppConfig._PARSER[account]['aws_secret'] for account in accounts_using_aws_secret], {})
+
+            # Download AWS Secrets
+            aws_client = boto3.client('secretsmanager')
```

Comment:
```suggestion
            try:
                aws_client = boto3.client('secretsmanager')
            except botocore.exceptions.BotoCoreError:
                Log.error('Unable to load AWS Secrets Manager - please make sure you have set up the AWS SDK for Python')
                return
```

Reply:
@simonrob I think your proposal here is related to your feedback regarding line 294:

> I'm a little reluctant to exit the proxy when errors or misconfigurations occur (both here and other places) unless they are absolutely critical. Perhaps just return here without loading?

In which case I'll update this code chunk based on the resolution to our conversation in that thread.

If I'm misinterpreting your intention here, let me know!